resotocore/resotocore/cli/command.py

```diff
@@ -2553,6 +2555,7 @@ def args_info(self) -> ArgsInfo:
         return [
             ArgInfo("--csv", help_text="format", option_group="format"),
             ArgInfo("--markdown", help_text="format", option_group="format"),
+            ArgInfo("--table", help_text="format", option_group="format"),
```

Comment:
Should this be `list --json` Since list already implies that we're only outputting the list view, just that instead of markdown or key=value it's represented as json?

Reply:
I was starting like this as well and thought that `table` describes the output better 
```json
{columns=[....]}
{row={...}}
{row={...}}
{row={...}}
```

I can also download csv or markdown in json, ndjson, xml, etc.
```
$> echo "search is(instance) | list --csv" | http :8980/cli/execute "Content-Type: text/plain" 
HTTP/1.1 200 OK
Content-Type: application/json

[   "\"kind\",\"id\",\"name\",\"instance_memory\",\"age\",\"last_update\",\"cloud\",\"account\",\"region\",\"zone\"",
    "\"kubernetes_node\",\"0c6d10e4-d9d5-458d-a561-f6d7c1b9cd47\",\"fixsaas-ccx23-pool-jobs-worker1\",15.2413330078125,\"14d10h\",\"14d10h\",\"k8s\",\"dev\",\"default\",\"\""
]
```

What about `--json-table`?

Reply:
I like `--json-table`. It's very clear what you get with it.